src/sketch.js

```diff
@@ -1,21 +1,28 @@
 /**
  * p5.js setup.
- * 
+ *
  * Returns:
  *  undefined.
  */
-function setup() 
-{
+let song;
```

Comment:
Please document this variable:

```suggestion
// soundtrack song container
let song;
```